enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/config/IProovConfig.java

```diff
@@ -67,4 +93,68 @@ public RestClient restClientIProov(IProovConfigProps configProps) throws RestCli
         return new DefaultRestClient(restClientConfiguration);
     }
 
+
+    @Bean
```

Comment:
Maybe too soon but we should qualify the bean in case we have more than one provider using OAuth 2.0...

Reply:
Qualifier is already used, `@Qualifier("iproovManagemenentWebClient") WebClient webClient) {` https://github.com/wultra/enrollment-server/pull/770/files#diff-2aa3940816a2a6154066d5fa9310d3f5e4e04b5dcb874f225992f34e1d5e4124R108

Reply:
I see, resolved then.